### ISSUE TITLE
iron-menu-behavior accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-menu-behavior#9]
Contrast: When I select one of the items in the "simple menu", item 2 becomes a very light color with poor contrast. Can we make that a slightly darker shade so that it's readable? 

Audit results: 
[Warning] Text elements should have a reasonable contrast ratio (12)
[Warning] These elements are focusable but either invisible or obscured by another element (4)
Not applicable tests (10)

**Comment by morethanreal** [comment]
Won't fix because this element doesn't implement UI. The demo element is only to show behavior.
